tests/providers/common/sql/hooks/test_sqlparse.py

```diff
@@ -24,16 +24,13 @@
     "line,parsed_statements",
     [
         ('SELECT * FROM table', ['SELECT * FROM table']),
-        ('SELECT * FROM table;', ['SELECT * FROM table']),
-        ('SELECT * FROM table; # comment', ['SELECT * FROM table']),
-        ('SELECT * FROM table; # comment;', ['SELECT * FROM table']),
-        (' SELECT * FROM table ; # comment;', ['SELECT * FROM table']),
-        ('SELECT * FROM table;;;;;', ['SELECT * FROM table']),
-        ('SELECT * FROM table;;# comment;;;', ['SELECT * FROM table']),
-        ('SELECT * FROM table;;# comment;; ;', ['SELECT * FROM table']),
```

Comment:
Why did I remove these test cases?

In the previous PR, some functionality was added that tried to make SQL query clearer.
But also it caused a few bugs. I think I tried to do some jobs for a developer and it was a bad idea.
I think we could add some functionality like `split_statements` but the developer should decide either to use it or not. But with semicolon stripping it wasn't that.